src/pages/Explore_Regions.py

```diff
@@ -119,54 +127,101 @@ def plot_comp(m_route, cause, df):
+
+st.markdown("The following is a table of all included migration routes paired with the total number of recorded dead and missing people from that route, since 2014.")
+dff = df.groupby(['Migration route'])[
+    ['Total Number of Dead and Missing']].sum().sort_values('Total Number of Dead and Missing', ascending=False).reset_index()
+st.dataframe(dff)
+
 st.sidebar.write("Data Filters")
+
 route_input = [st.sidebar.selectbox(
     'Migration Route', df['Migration route'].unique().tolist())]
+st.header('Exploring the '+route_input[0])
+
+
+if route_input:
+    route_s = route_input[0]
+    mdf = df[df['Migration route'] == route_s]
+    migrantdf = mdf.groupby(['Cause of Death'])[
+        'Total Number of Dead and Missing'].sum().reset_index().sort_values(by='Total Number of Dead and Missing', ascending=False)
+    route_s = route_input[0]
+    worst_month = mdf.groupby(['Migration route', 'date'])['Total Number of Dead and Missing'].sum(
+    ).reset_index().sort_values(by='Total Number of Dead and Missing', ascending=False)
+    df_d_s = mdf.groupby(['Migration route'])[
+        ['Total Number of Dead and Missing', 'Number of Survivors', 'Number of Females', 'Number of Males', 'Number of Children']].sum().reset_index()
+
+    total_dead_missing = prettify(
+        df_d_s['Total Number of Dead and Missing'].iloc[0])
+    total_women_dead_missing = prettify(df_d_s['Number of Females'].iloc[0])
+    total_men_dead_missing = prettify(df_d_s['Number of Males'].iloc[0])
+    total_children_dead_missing = prettify(
+        df_d_s['Number of Children'].iloc[0])
+
+    # introduction
+    st.markdown("For the " + route_input[0] +
+                " migration route, there are a total of  " + total_dead_missing +
+                " recorded deaths in the data since 2014. This includes " + total_women_dead_missing +
+                " women, " + total_men_dead_missing + " men, and " + total_children_dead_missing + " children.")
+
+    # map of the selection region
+    # TODO: there's some data that shows up that shouldn't be here..
+    st.map(mdf)
+
+    # Country of origin
+    mdf['Number of People'] = mdf.loc[:, [
+        'Total Number of Dead and Missing', 'Number of Survivors']].sum(axis=1)
+    total_migrants = mdf['Number of People'].sum()
+    mdf['Percentage of People from Country of Origin'] = mdf['Number of People']/total_migrants
+    dforigin = mdf.groupby(['Country of Origin'])[['Number of People', 'Percentage of People from Country of Origin']].sum(
+    ).sort_values('Number of People', ascending=False).reset_index()
+
+    st.markdown("The following table lists the known countries of origin for migrants traveling along this route. For each country, we see the total number of people from the country and the percentage of migrants from the country.")
+    st.dataframe(dforigin)
```

Comment:
Country of origin data! displayed as a table. Might be interesting to display as a heat map or something